tests/Entity/OrderTest.php

```diff
@@ -397,4 +397,28 @@ public function testCanUnlock()
         $orderHistory->setType('active');
         $this->assertSame(true, $order->canUnlock());
     }
+
+    public function testOrderTypeDisplayText()
```

Comment:
You could use a [data provider](https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#data-providers) to both simplify this (and make it easier to add additional test cases) as well as make it easier to understand what the test is doing.